### PR TITLE
#18 Settings Page - frontend form

### PR DESCRIPTION
**Pull request by SPTLAbron**
I completed the front-end code for both the user and admin settings pages. I used a separate file for the edit button.

### CODE REVIEW

**Comment by RickyLi15** [comment]
In both the UserPage and the AdminPage under the settings folder, can you change "import classes from..." to "import styles from..." just for uniformity?

**Comment by RickyLi15** [comment]
Also, can you rename your setting pages to UserSettingsPage instead of just UserPage? Same thing for AdminPage.

**Review by RickyLi15 — COMMENTED** [review]
Please move pen.png into the assets folder. I left some other comments as well.

**Comment by SPTLAbron** [comment]
Ignore the not all commit. I made the requested changes and fixed the style names to be camel case. I also used the figma svg for the pencil

File: jaba/src/assets/pen.png
Comment (yieb) [inline jaba/src/assets/pen.png:1]: Is there a reason why you're using both a PNG and SVG of the same image?

**Comment by yieb** [comment]
image

When I zoom in a lot, the "Edit Category" button starts to overlap with the input boxes. Can you maybe look into fixing this styling issue?

**Comment by yieb** [comment]
image

When the "Delete" button is clicked, a popup should appear. Can you update your PR to include this change? I would suggest maybe looking into using something like MUI's Modal component (https://mui.com/material-ui/react-modal/). Or you can take a look at how Krushi, Andrew, and Megan did this with a modal in the RequestAccount directory for an example of how to do this if this is easier.

**Comment by yieb** [comment]
Merging in now since it looks good on the user side and matches design. Since the User and Admin Page Settings Pages are nearly identical we should combine the two files into one and use a single CSS module for styling later. Also styling can be improved to improve page layout on zoom in/out.

**Review by yieb — APPROVED** [review]
Merging in now since it looks good on the user side and matches design. Since the User and Admin Page Settings Pages are nearly identical we should combine the two files into one and use a single CSS module for styling later and add some consistency between the popup on this page and the ones on other pages. Also styling can be improved to improve page layout on zoom in/out.

**Pull request merged**